src/test/java/nl/praegus/fitnesse/responders/ToolTip/ToolTipsTest.java

```diff
@@ -0,0 +1,56 @@
+package nl.praegus.fitnesse.responders.ToolTip;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ToolTipsTest {
+
+
+    @Test
+    public void check_if_ToolTip_is_not_null_when_fixture_tooltips_and_bootstrap_tooltips_are_given() {
+        ToolTips toolTips = new ToolTips(System.getProperty("user.dir") + "/src/test/resources/Tooltip/TooltipData_default", System.getProperty("user.dir") + "/src/test/resources/Tooltip/TooltipData_default/toolchain.jar");
+
+        String receivedResult = toolTips.getRandomToolTip();
+
+        assertThat(receivedResult).isNotEqualTo(null);
+    }
+
+    @Test
+    public void Check_if_ToolTip_Is_null_when_Nothing_is_given() {
```

Comment:
nothing is given? wanneer de tooltipfiles leeg zijn of zo?

Reply:
deze is voor als de tooltip txt files worden gevonden maar leeg zijn, ik heb deze even anders genoemd voor de duidelijkheid